tests/components/tesla/test_config_flow.py

```diff
@@ -0,0 +1,104 @@
+"""Test the Tesla config flow."""
+from unittest.mock import Mock, patch
+
+from tests.common import mock_coro
+
+from homeassistant import config_entries, data_entry_flow, setup
+from homeassistant.components.tesla import config_flow
+from homeassistant.components.tesla.const import DOMAIN
+from homeassistant.const import (
+    CONF_ACCESS_TOKEN,
+    CONF_PASSWORD,
+    CONF_TOKEN,
+    CONF_USERNAME,
+)
+from teslajsonpy import TeslaException
```

Comment:
Please sort and group imports. Move this up between standard library and the homeassistant import groups.